src/manager/image.manager.ts

```diff
@@ -24,6 +24,12 @@ export class ImageManager {
 		return this.imageDatastore.getAll();
 	}
 
+	public async getImageById(id: string): Promise<Image> {
+		
+		const image: Image = await this.imageDatastore.getById(id);
```

Comment:
return this.imageDatastore.getById(id)

you don't have to wait for the response in this case